### PR TITLE
feat(randomizelogo): Added different logoes and rotation of logo display

### PR DESCRIPTION
**Pull request by Magjerds**

### CODE REVIEW

File: src/containers/MainLayout.tsx [hunk 1]
```diff
@@ -47,6 +55,28 @@ const MainLayout: React.FC<MainLayoutProps> = ({ children }) => {
...
+    const index = Math.floor(Math.random() * logoes.length)
+    return logoes[index]
+  }
+  const [logoUrl, setLogoUrl] = React.useState(logoes[0])
+  React.useEffect(() => {
+    const intervalId = setInterval(() => {
+      setLogoUrl(randomizeLogo()) // Update logoUrl every 5 seconds
+    }, 5000) // 5000 milliseconds = 5 seconds
+
+    return () => clearInterval(intervalId) // Cleanup the interval on component unmount
```
Comment (alexaor) [inline src/containers/MainLayout.tsx:78]: Comment is redundant

```suggestion
    return () => clearInterval(intervalId)
```

File: src/containers/MainLayout.tsx [hunk 2]
```diff
@@ -47,6 +55,28 @@ const MainLayout: React.FC<MainLayoutProps> = ({ children }) => {
...
+    logoUrl4,
+    logoUrl5,
+    logoUrl6,
+    logoUrl7,
+  ]
+  function randomizeLogo() {
+    const index = Math.floor(Math.random() * logoes.length)
+    return logoes[index]
+  }
+  const [logoUrl, setLogoUrl] = React.useState(logoes[0])
```
Comment (alexaor) [inline src/containers/MainLayout.tsx:72]: Import `useState` and move up together with other hooks
```suggestion
  const [logoUrl, setLogoUrl] = useState(logoes[0])
```

**Review by alexaor — CHANGES_REQUESTED** [review]
Fun stuff!

Up to 2MB per asset file is sadly too large to ship as a build. Logo type assets should maximum be 200kb.

Should probably also be deleted from from git history altogether so not everyone has to download these assets whenever they clone the repo. I think local squash + force push should clean it up

Other than that most comments are just to follow our "style guide", not that it is formalised anywhere.

**Comment by Plebbimon** [comment]
Additionally, maybe we could look at directing the images to the static file dir that django serves (in the same way it serves a profile picture with `/media/profiles/{id}`). I think the approach works for now, but in case we want more pictures and avoid filling up the assets frontend

**Comment by alexaor** [comment]
"technically" we serve static files using apache

**Comment by alexaor** [comment]
but the app is hosted in a bucket and distributrd with cloudfront cdn. So storing it with the bundle isnt really an issue. It just creates another implicit dependency.

Would make more sense to move to the backend if we want users to control logo assets (like each internal group can upload their own logo) in a white label type solution

**Pull request closed without merging**